### DISCUSSION TITLE
Tell component section to truncate from right to left rather than vice versa

### DISCUSSION THREAD

**Discussion by mawkler** [famiu/feline.nvim#212] [Q&A] [answered]
By default Neovim seems to truncate the statusline from left to right if it doesn't fit. I like to have my Vim mode in the bottom left, but it therefore the first thing that disappears upon truncation. I would like for it to be the very last thing in that component section to dissappear.

I found `%<` in `:h statusline` but there doesn't seem to be a `%>`.

I know of #96 which sort of covers this issue, but it has a much wider scope.

I also know that I can use `enabled` to adapt my components based on `fn.winwidth()`, but sometimes the statusline still becomes too wide.

Is this possible to do?

**Comment by ram02z** [comment, marked as answer]
> 
> Is this possible to do?


By default components have a truncation priority of `0`, and you can try to set a higher priority to the vim mode component, say `1`.

You can also set `truncate_hide = true` to components you want to hide first. 

See https://github.com/feline-nvim/feline.nvim/blob/master/USAGE.md#component-priority

**Reply by mawkler** [reply]
Interesting, I must have missed that. Will check it out, thank you! :)
